README.md

```diff
@@ -76,7 +76,7 @@ See the documentation for the latest releases:
 
 **MongooseIM documentation highlights:**
 
-When developing new features/modules, please make sure you add basic documentation to the 'doc/' directory, and add a link to your document in 'doc/README.md.'
+When developing new features/modules, please make sure you add basic documentation to the ['doc/'](https://github.com/esl/MongooseIM/tree/master/doc) directory, and add a link to your document in ['mkdocs.yml'](https://github.com/esl/MongooseIM/blob/master/mkdocs.yml).
```

Comment:
```suggestion
When developing new features/modules, please make sure you add basic documentation to the ['doc'](doc) directory, and add a link to your document in ['mkdocs.yml'](mkdocs.yml).
```

Reply:
Ah, true, it's from GH to GH - too much looking at the docs.